src/lib/connection/connector/fallback.js

```diff
@@ -55,6 +55,8 @@ async function sendMessage(cmd, payload) {
       return sendValue(fallbacks.isAndroid());
     case action.openLink:
       return sendValue(fallbacks.openLink.apply(payload));
+    case action.nativeUpload:
+      return sendValue(fallbacks.nativeUpload());
```

Comment:
fallbacks.nativeUpload() returns void and then is "bind to the connect promise" as there is nothing we can do shouldn't we also skip this?

Reply:
would be an exceptional case compared to all others, i think uniformity is better